framework/doc/content/source/postprocessors/FunctionElementAverage.md

```diff
@@ -0,0 +1,13 @@
+# FunctionElementAverage
```

Comment:
already exists
https://mooseframework.inl.gov/source/postprocessors/FunctionElementAverage.html ?

Reply:
this might be a relatively recent addition

Reply:
oh you have two commits on the PRs that you do not need

Reply:
Not sure how these got in?

Reply:
did you `git rebase -i HEAD~ too_far`? It can happen that way, basically you change the author of the commits that were just before your work so they count as new commits